util/customTransformer-py.js

```diff
@@ -0,0 +1,58 @@
+const stats = require("./stats");
+const { getMetadata } = require("../v0/util");
+const { invokeLambda, testLambda } = require("./lambda");
+const { JSON_IMPORT_CODE, TRANSFORM_WRAPPER_CODE } = require("./lambda/utils");
+
+async function pyUserTransformHandler(
+  events,
+  userTransformation,
+  testMode = false,
+  testWithPublish = false
+) {
+  if (userTransformation.versionId) {
+    const metaTags =
+      events.length && events[0].metadata
+        ? getMetadata(events[0].metadata)
+        : {};
+    const tags = {
+      transformerVersionId: userTransformation.versionId,
+      language: userTransformation.language,
+      version: 1,
+      ...metaTags
+    };
+
+    const transformationPayload = {
+      events,
+      transformationType: "transformEvent"
+    };
+
+    let result;
+    if (testMode) {
+      const lambdaCode =
+        JSON_IMPORT_CODE + userTransformation.code + TRANSFORM_WRAPPER_CODE;
```

Comment:
Why can't we have the code to split in different python files and then use it using import, it is bit clean right

Reply:
It is more or less executing two functions transformEvent & transformWrapper. It should be fine having them in one file. Convenient while creating/reading zip too.

Reply:
added in separate files